src/Components/FrontPage.js

```diff
@@ -1,32 +1,44 @@
 import { useState } from "react"
 import TagLines from "./TagLines"
+import "./FrontPage.css"
+import { useNavigate } from "react-router-dom"
 
 function FrontPage( {setUserEmail} ) {
+    const [name, setName] = useState("")
```

Comment:
These two setstates shouldn't be used if you're passing them down from props

Reply:
The set name is to control the input

Reply:
setUserName is to pass the info between components